src/components/resources/index.js

```diff
@@ -129,13 +129,13 @@ const RESOURCES = [
         title: "Wallet SDK Swift",
         Svg: Graphics.walletSdkSwift,
         content: "Build Swift-based applications using our SDKs",
-        linkTo: "https://input-output-hk.github.io/atala-prism-wallet-sdk-swift/"
+        linkTo: "https://input-output-hk.github.io/atala-prism-wallet-sdk-swift/documentation/atalaprismsdk/"
     },
     {
         title: "Wallet SDK TypeScript",
```

Comment:
to `Edge Agent SDK TypeScript` or `Edge Agent SDK TS`

Reply:
fixed